package.json

```diff
@@ -93,17 +101,10 @@
     "ts-jest": "^27.0.7",
     "tsup": "^5.10.0",
     "typescript": "^4.5.2",
+    "unplugin-vue-components": "^0.17.2",
```

Comment:
Can we avoid directly relying on it?

Reply:
But we have to import its type `ComponentResolver`

Reply:
Or we just make it optional?